### PR TITLE
Added documentation to convert this boilerplate into electron desktop…

### PR DESCRIPTION
**Pull request by mjangir**
I've added a complete documentation to create an electron desktop application out of this awesome boilerplate. Though I've not changed anything in source code as that would requires lots of webpack configuration changes and files addition that will not allow to run the application in web browser.

## React Boilerplate

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [ ] You have followed our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
- [ ] double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Internal code generators and templates are updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/react-boilerplate/react-boilerplate/blob/master/LICENSE.md).

### CODE REVIEW

**Comment by julienben** [comment]
Wow lots of stuff here! Thanks for the contribution. I'm currently traveling but will be happy to review when I get back.

File: docs/README.md [hunk 1]
```diff
@@ -34,6 +34,16 @@
   - [routing](js/routing.md)
 - [Maintenance](maintenance)
   - [Dependency Update](maintenance/dependency.md)
+- [Convert React Boilerplate To Electron App](electron-version)
```
Comment (julienben) [inline docs/README.md]: The main readme terms of content shouldn't have 10 lines dedicated to the electron documentation.

**Comment by julienben** [comment]
There's so many instructions and code here. Im slightly worried that this will need to be heavily maintained or it might become obsolete as the rest of the project evolves. Curious to hear what other maintainers think?

File: docs/README.md (hunk 1 above)
Comment (julienben) [reply docs/README.md]: One line is enough. The main electron readme file has its own toc.

**Comment by mjangir** [comment]
@julienben whenever the project evolves, these instructions shouldn't break or affect anything as they are completely isolated. The electron documentation is much focusing on correct guidelines rather than a huge change in source code.

**Comment by gretzky** [comment]
Great write up @mjangir. I also agree that it seems a bit heavy, but that being said - I recommend moving the write up you've done here to a separate README / repo and then in the rbp docs we can link to it. Thoughts?

**Comment by mjangir** [comment]
@gretzky That's also fantastic. I can put those files in one of my GitHub repo and can provide the link.

**Comment by gretzky** [comment]
Great @mjangir! Only one last change, please. Can you change the base branch of this PR from react-boilerplate:master to react-boilerplate:dev?

**Comment by julienben** [comment]
Closing as this has been merged in #2358

**Pull request closed without merging**